test/test_checksums.py

```diff
@@ -80,11 +81,13 @@ def test_crc32c_large_buffer(self):
         self.assertEqual(0xfb5b991d, val)
 
     def test_crc32c_huge_buffer(self):
-        # stress the internal logic that handles buffers larger than C's INT_MAX
+        if sys.platform.startswith('freebsd'):
```

Comment:
only fails for crc32c? do we want to skip other flex checksum cases for huge buffer?

Reply:
It only fails for crc32c for now, I skip the other huge_buffer test for safe.